msvc-full-features/Cataclysm-vcpkg-static.vcxproj

```diff
@@ -242,4 +242,4 @@
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <ImportGroup Label="ExtensionTargets">
   </ImportGroup>
-</Project>
+</Project>
```

Comment:
```suggestion
</Project>

```

Reply:
I messed up and I am unable to accept it, but I made the changes myself so it should be good.